RusimbiPatrick/Banka
# #165310870 built staff activate and deactivate bank account

**What does this PR do?**

It helps an admin to update an account status by activating or deactivating it.

**Description of Task to be done?**

1. 
2. It adds a controller for activating and deactivating.
3. It adds validations for account number and status input.
4. It adds a service for for activating and deactivating.
5. It adds account update route on account router

**How should this be manually tested?**

Start the Server and POSTMAN.
Browse the following endpoint route: localhost:3000/api/v1/accounts/account-number.
Submit the required information with PATCH method.
Check if you receive a message telling you that Account ${validStatus}d succesfully
